src/os/bluestore/BitAllocator.h

```diff
@@ -364,6 +366,7 @@ class BitMapZone: public BitMapArea{
   }
 
   int64_t alloc_blocks(int64_t num_blocks, int64_t *start_block);
+  using BitMapArea::alloc_blocks_dis;
```

Comment:
I have two remarks regarding alloc_blocks_dis method(s):
The first one - I suggest to avoid overload virtuals at all. Hence one can rename alloc_blocks_dis( bool wait...) to something like alloc_blocks_dis_wait..
The second - Probably I missed something but I'm unable to find any use of this bool wait parameter in alloc_blocks[_dis[_int]] methods. I can see they are passed between these methods but don't see real wait var handling. Could you please point out that to me. And if my finding is correct probably one can simplify these methods and get rid of some of them...